### DISCUSSION TITLE
Adding closing x button.

### DISCUSSION THREAD

**Discussion by bomshteyn** [spatie/laravel-support-bubble#25] [Ideas]
My client has asked me to add a closing x button to the top corner. Like in the attached screenshot.
![screenshot](https://user-images.githubusercontent.com/4259699/149016999-328147ee-bee4-44b6-a3da-319971502f96.png)

I have achieved it by overriding the `support-bubble.blade.php` & `script.blade.php`. If others are interested in having this as well i can create a PR.

**Comment by colinmackinlay** [comment]
Interested to see this. I would also suggest modifying it so that clicking away from the form also closes it instead of only being able to  close by clicking the bubble again
